synapse/tests/test_cortex.py

```diff
@@ -4847,3 +4817,59 @@ async def test_cortex_vars(self):
                 self.eq('hoho', await proxy.getStormVar('lolz', default='hoho'))
                 self.eq('haha', await proxy.popStormVar('hehe'))
                 self.eq('hoho', await proxy.popStormVar('lolz', default='hoho'))
+
+    async def test_cortex_syncnodefilterededits(self):
+        async with self.getTestCore() as core:
+            items = await alist(core.syncNodeFilteredEdits(0, {}, wait=False))
```

Comment:
Tests should be run against telepath exposed APIs to make sure they work

Reply:
ok